tabular/src/autogluon/tabular/task/tabular_prediction/predictor.py

```diff
@@ -550,9 +550,8 @@ def transform_labels(self, labels, inverse=False, proba=False):
                 labels_transformed = self._learner.label_cleaner.transform(y=labels)
         return labels_transformed
 
-    # TODO: Consider adding time_limit option to early stop the feature importance process
     # TODO: Add option to specify list of features within features list, to check importances of groups of features. Make tuple to specify new feature name associated with group.
-    def feature_importance(self, dataset=None, model=None, features=None, feature_stage='original', subsample_size=1000, silent=False, **kwargs):
+    def feature_importance(self, dataset=None, model=None, features=None, feature_stage='original', subsample_size=1000, time_limit=None, num_shuffle_sets=None, silent=False):
         """
         Calculates feature importance scores for the given model.
```

Comment:
Add tutorial link here too (https://explained.ai/rf-importance/). Have gotten many questions about this previously.

Reply:
Added